Objects/longobject.c

```diff
@@ -4666,6 +4668,9 @@ long_bitwise(PyLongObject *a,
         break;
     default:
         PyErr_BadArgument();
```

Comment:
Ditto: use "Py_UNREACHABLE();" instead. No need to both with references. It's deadcode.